test-requirements.txt

```diff
@@ -5,3 +5,10 @@ pep8==1.5.7
 pyflakes==0.8.1
 mccabe==0.2.1 # capped for flake8
 bashate>=0.2 # Apache-2.0
+
+
+
+# this is required for the docs build jobs
+sphinx!=1.2.0,!=1.3b1,<1.3,>=1.1.2
```

Comment:
Does RPC-O require this rather complex list of acceptable versions? I know we need it upstream, but for our other RPC docs deconst seems quite happy with sphinx>=1.3.4